src/trees/MWTree.h

```diff
@@ -103,6 +103,9 @@ template <int D> class MWTree {
     void makeNodeTable(MWNodeVector<D> &nodeTable);
     void makeNodeTable(std::vector<MWNodeVector<D>> &nodeTable);
 
+    int getNodeTableSize() const { return this->endNodeTable.size(); }
```

Comment:
I think we already have `getNEndNodes()` ?

Reply:
Oh god yes, didn't see that!